test/inductor/test_cudagraph_trees.py

```diff
@@ -387,6 +387,21 @@ def foo(x):
             # and then again to record it
             self.assertEqual(self.num_checkpoints(), 2)
 
+        def test_expanded_inputs(self):
+            x = torch.empty_strided([4, 512], [0, 1], device="cuda")
+            for i in range(4):
+                x[i].random_(0, 1)
```

Comment:
but why?

Reply:
Did not see rand_strided api…

Reply:
yeah but you still don't need a loop, it's an expanded tensor, so either do `x = torch.rand(1, 512, device="cuda").expand(4, 512)`, or `empty_strided` like you have and `x[0].random_(0,1)`. Anyway, it doesn't matter, just a bit confusing the way it's written.

Reply:
I did it this way because the logic of the PR is that we only need to access one element from the tensor, so if we did it in the test too that would have been kind of circular logic (even though we know it's correct)

Reply:
But it's not really part of the test, it's preparing inputs for the test, doesn't matter how you do it as long as inputs are what you want (a tensor with the needed content and metadata)